tests/test-repo.c

```diff
@@ -249,6 +249,29 @@ test_write_regfile_api (Fixture *fixture,
   g_assert_cmpstr (checksum, ==, "23a2e97d21d960ac7a4e39a8721b1baff7b213e00e5e5641334f50506012fcff");
 }
 
+/* Just a sanity check of the C autolocking API */
+static void
+test_repo_autolock (Fixture *fixture,
+                        gconstpointer test_data)
```

Comment:
Nitpick: Indentation is off here.

Also ‘sanity’ in the comment above could be considered a bit ableist. How about “Basic check of the C autolocking API”?